### PR TITLE
Update FCM specs to match new FCM styles

### PR DESCRIPTION
**Pull request by niqjohnson**
Update the FCM documentation to match the new FCM styles that went into production a few weeks ago.

## Additions

- Added specs for play button on video FCMs

## Removals

- Removed specs for the call to action button, which is now no longer an option in FCMs
- Removed all mention of and specs for the category slugs

## Changes

- Updated visual spcs
- Updated character counts
- Updated FCM screenshots at all three sizes

## Testing

If you have the Design Manual running locally, pull this branch and visit the FCM page under "Page components". If you don't, take a look at the screenshot below.

When reviewing, please pay particular attention to:

- The updated visual style specs. I'm pretty sure I have all the typography, color, and size specs matching the new FCM, but I was pulling them from a lot of sources, so some extra eyes on those would be appreciated.
- The description of how the image size and aspect ratio changes with screen size. I'm not 100% sure if how I'm describing it here makes sense.
- The updated character counts to make sure they're what we agreed on.

## Screenshots

dm-fcm

## Todos

- Once this is approved and merged, I'll update all the other mentions of the FCM in our documentation to match these specs.

### CODE REVIEW

File: page-components/featured-content-module.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ redirect_from: "/ui-toolkit/featured-content-module.html"
 ---
 
 - [Use cases](#use)
-- [Desktop style](#desktop)
-- [Tablet style](#tablet)
-- [Mobile style](#mobile)
+- [Desktop style](#style-at-desktop-size)
+- [Tablet style](#style-at-tablet-size)
+- [Mobile style](#style-at-mobile-size)
```
Comment (Scotchester) [inline page-components/featured-content-module.md:12]: Good catch!

File: page-components/featured-content-module.md [hunk 2]
```diff
@@ -39,61 +39,45 @@ Featured content modules (FCMs) function to highlight a specific piece of conten
...
 </div>
 
 <div class="content-33 content-last"></div>
 
 ## Style at desktop size
 
+{::nomarkdown}
 <div>
-![Image of featured content module example at desktop size]({{ site.baseurl }}/static/img/featured-content-module/fcm-desktop02.png)
+  <img src="{{ site.baseurl }}/static/img/featured-content-module/fcm-desktop02.png" alt="Image of featured content module example at desktop size">
```
Comment (Scotchester) [inline page-components/featured-content-module.md:58]: Did you do this to get it out of the `<p>` tag that was constraining its width? Smart!

File: page-components/featured-content-module.md [hunk 3]
```diff
@@ -39,61 +39,45 @@ Featured content modules (FCMs) function to highlight a specific piece of conten
...
-* Module is a minimum of 320px total in height with 30px top and bottom padding. It appears 60px underneath the header/intro paragraph of the page.
-* Background color of module is Gray 5 with a 1px Gray 50 stroke.
-* Module bleeds to the edge of the page, right sidebar and left nav.
-* Visual spans 5 columns and bleeds into right sidebar. The visual is sized at 640 x 360px (a 16:9 ratio) for optimal appearance at all screen sizes.
-* Text spans 3 columns and is left and top aligned.
+<div class="content-50 content-first">
+* Module spans the full width of the page’s content area and is a minimum of 220px tall. It appears 60px underneath the header/intro paragraph of the page.
+* Background color of module is Gray 5 with a 1px Gray 40 stroke.
+* Visual is 270px wide and a minimum of 220px tall. The height of the visual will vary at different screen sizes so that the full height of the image is always visible. The aspect ratio of the image changes with screen size to keep the full image height visible; the image will be cropped up to 23% from each side.
+* Text spans 5 columns and is left and top aligned. There is 30px of padding around all sides of the text.
```
Comment (Scotchester) [inline page-components/featured-content-module.md]: Might want to do this.

```suggestion
* Text spans 5 columns (at maximum page width) and is left and top aligned. There is 30px of padding around all sides of the text. As available width dips below the maximum page width, the text area shrinks in width while the visual maintains a fixed width of 270px.
```

After writing that, I wonder if it'd be better take the responsive behavior mentions out of both above bullets and add a bullet dedicated to explaining how the layout shifts below maximum page width. Thoughts?

File: page-components/featured-content-module.md [hunk 4]
```diff
@@ -39,61 +39,45 @@ Featured content modules (FCMs) function to highlight a specific piece of conten
...
-  * <h4>{% include icons/play-round.svg %} Featured video</h4>
-  * <h4>{% include icons/settings.svg %} Featured tool</h4>
-  * <h4>{% include icons/newspaper.svg %} Featured news</h4>
-* If the content does not fit within one of the five categories, you may use this icon and prefix:
-  * <h4>{% include icons/favorite.svg %} Featured </h4>
+* Call to action link: 16px, standard link colors, appears 15px below body copy, 35 maximum character count (including spaces)
+* Play button (for FCMs with a video): 60px diameter, Black background at 75% opacity that changes to Pacific on hover, White icon, 2px Gray 5 border, centered in the visual
 </div>
 
 ## Style at tablet size
```
Comment (Scotchester) [inline page-components/featured-content-module.md]: Should we just get rid of this section? It doesn't seem to serve much purpose anymore.

File: page-components/featured-content-module.md [hunk 5]
```diff
@@ -110,11 +94,11 @@ Differences in style at tablet size
 
 Differences in style at mobile size
 
-* Module appears 30px underneath the header/intro paragraph of the page
-* Headline: H3
-* Body copy: 16px
+* Height is variable depending on how much content is in the FCM
```
Comment (Scotchester) [inline page-components/featured-content-module.md]: This is not different from the wider layout, and would probably be assumed by most, anyway.

File: page-components/featured-content-module.md [hunk 6]
```diff
@@ -39,61 +39,45 @@ Featured content modules (FCMs) function to highlight a specific piece of conten
 
 <div class="content-67 content-first">
 
-Featured content modules should include a single piece of content from a defined content type, including: Blog, News, Video, Event, and Tool.
+Featured content modules should highlight a single piece of content. That content can be almost anything, from a webpage to a blog post to a PDF to a video.
 
 FCMs appear below the title and introductory paragraph on a page, and retain that position on all screen sizes.
 
 FCMs contain a headline, body copy text, and a graphical element related to the piece of content being featured. They are a “mini story” and never simply a dominant visual.
```
Comment (jenn-franklin) [inline page-components/featured-content-module.md]: Should we add "call to action link" to this description? Maybe, "FCMs contain a headline, body copy text, call to action link if applicable, and graphical element related to the piece of content being featured."

**Review by jenn-franklin — APPROVED** [review]

File: page-components/featured-content-module.md (hunk 2 above)
Comment (niqjohnson) [reply page-components/featured-content-module.md:58]: Yep, that seemed to be the only way I could get the image to be big enough to see well.

File: page-components/featured-content-module.md (hunk 3 above)
Comment (niqjohnson) [reply page-components/featured-content-module.md]: I reworked this in eeb539d. Splitting out the responsive behavior into a separate bullet seemed like the way to go to me, and I think this section reads better now.

File: page-components/featured-content-module.md (hunk 4 above)
Comment (niqjohnson) [reply page-components/featured-content-module.md]: Agreed. I combined the desktop and tablet specs in eeb539d.

File: page-components/featured-content-module.md (hunk 5 above)
Comment (niqjohnson) [reply page-components/featured-content-module.md]: Agreed. I reworked this a little in eeb539d to put all the info about text and visual stacking in one bullet.

**Comment by niqjohnson** [comment]
Thanks for taking a look, @huetingj and @Scotchester. I pushed an update that incorporates your feedback:

fcm-updates

Let me know if you see any other changes.

**Pull request merged**